### ISSUE TITLE
Not working with new Cypress 11.0.0 

### ISSUE THREAD

**Issue by matifniaz** [badeball/cypress-cucumber-preprocessor#877]
Not able to install @badeball/cypress-cucumber-preprocessor

**Version : [13.1.0](https://www.npmjs.com/package/@badeball/cypress-cucumber-preprocessor/v/13.1.0)
OS : windows 11
Cypress : 11**

The snapshot is attached for reference.

![image](https://user-images.githubusercontent.com/40147842/200734926-0762197c-c0e9-4129-8831-7bf60d7079ef.png)

**Comment by k2kirov** [comment]
I'd like for this to be resolved as well. Thanks!

**Comment by dmaicher** [comment]
@badeball I think it might just be a matter of allowing `^11.0.0` as well as peer dep? I tried npm installing using `--force` on my app using Cypress 11.0 and it works just fine :blush:

**Comment by matmannion** [comment]
You can avoid using `--force` by using the `overrides` key in your package.json:

```
  "overrides": {
    "@badeball/cypress-cucumber-preprocessor": {
      "cypress": "$cypress"
    }
  },
```

This will set the peer `cypress` dependency to the same version as the `cypress` dependency in your own `package.json`, at least until a version of the library is released that declares it supports cypress 11

**Comment by badeball** [comment]
Fixed with v14.0.0.
